framework/test/mocha/unit/modules/chain/chain.js

```diff
@@ -449,14 +449,14 @@ describe('Chain', () => {
 			chain.options.syncing.active = false;
 
 			// Act
-			chain._startLoader();
+			await chain._startLoader();
 
 			// Assert
 			expect(stubs.jobsQueue.register).to.not.be.called;
 		});
 
 		it('should load transactions and signatures', async () => {
-			chain._startLoader();
+			await chain._startLoader();
```

Comment:
Why did you re-introduce this wrapper?

Reply:
the function call couldn't be spied, so I moved it back. and also I checked the whole test suite, for all the channel events where we are subscribing or publish we do not have such tests, so moved it back, maybe we can clean this unwanted private functions when we have a way to test. For this PR I have addressed the missing await for that function.